packages/mdc-drawer/standard/mdc-standard-drawer.scss

```diff
@@ -0,0 +1,99 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "@material/rtl/mixins";
+@import "@material/typography/mixins";
+@import "@material/typography/variables";
+@import "@material/theme/variables";
+@import "@material/theme/mixins";
+@import "../variables";
+@import "../mixins";
+
+$mdc-drawer-width: 256px;
+$mdc-drawer-list-item-spacing: 4px;
+$mdc-drawer-surface-padding: 16px;
+
+.mdc-drawer {
+  @include mdc-drawer-border-color($mdc-drawer-divider-color, $mdc-drawer-divider-opacity);
+  @include mdc-drawer-group-header-text-color($mdc-drawer-subtitle-ink-color, $mdc-drawer-subtitle-ink-opacity);
+  @include mdc-drawer-surface-fill-color($mdc-drawer-surface-fill-color, $mdc-drawer-surface-fill-opacity);
+  @include mdc-drawer-icon-ink-color($mdc-drawer-icon-ink-color, $mdc-drawer-icon-ink-opacity);
+  @include mdc-drawer-meta-ink-color($mdc-drawer-icon-ink-color, $mdc-drawer-icon-ink-opacity);
+  @include mdc-drawer-item-text-color($mdc-drawer-inactive-ink-color, $mdc-drawer-inactive-ink-opacity);
+  @include mdc-drawer-item-activated-text-color($mdc-drawer-activated-ink-color, $mdc-drawer-activated-ink-opacity);
+
+  width: $mdc-drawer-width;
+  min-width: $mdc-drawer-width;
+  border-right-width: 1px;
+  border-right-style: solid;
+  overflow: hidden;
+
+  @include mdc-rtl {
+    border-right-width: 0;
+    border-left-width: 1px;
+    border-right-style: none;
+    border-left-style: solid;
+  }
+
+  .mdc-list,
+  .mdc-list-group .mdc-list {
+    padding: $mdc-drawer-surface-padding/2;
+    padding-top: 0;
+  }
+
+  // stylelint-disable selector-no-qualifying-type
+  .mdc-list-item,
+  a.mdc-list-item {
+    @include mdc-typography(body2);
+
+    height: calc(48px - 2 * #{$mdc-drawer-list-item-spacing});
+    margin: $mdc-drawer-list-item-spacing 0;
+    padding: 0 $mdc-drawer-surface-padding/2;
+  }
+  // stylelint-enable selector-no-qualifying-type
+
+  .mdc-list-group__subheader {
```

Comment:
Also, this should really be called subtitle, not subheader, according to the design spec. Maybe subtitle within header is just a specialization of this subtitle?

Reply:
Spec has 13dp for the label, but the params spreadsheet state 14dp (both subtitle2 and body2). I changed from subtitle2 --> body2 and looks closer. material.io spec doesn't explicitly call subtitle or subheader out. The "subtitle" that is called out in spec is referring to the primary text of the list item.

Reply:
RE list vs drawer stylesheet: I had the same question to design. They stated nav drawer list styles will only apply to nav drawer. Since its not specifically called out in list's spec, I believe its left open to the end designer/developer.

Reply:
I was referring to this wording:

![image](https://user-images.githubusercontent.com/398379/41184510-b416b63a-6b4e-11e8-8207-30ff03c8801f.png)

Reply:
Ah gotcha - I do have every reference in drawer as 'subtitle'. I can change subheader --> subtitle in list. It will be a breaking change.